news/2021-07-26-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,133 @@
+- **Speed**, which considers how quickly new elements are presented to the player during the course of the map (affected by things like the number of hit objects over time, BPM, Double Time and Half Time)
+- **Accuracy**, which considers the player's ability to stay on time rhythmically and how difficult this is to achieve (affected by things like Hidden, Hard Rock, Flashlight and more)
+- **Strain**, which considers how often and how long a player has to endure intense or technically demanding patterns during play (affected by things like streams, fast jumps and more)
+
+## Release Schedule
+
+## Star Rating
+
+### osu!: Nerfed maps with short difficulty spikes
+
+To understand this change, first let's quickly recap how *skills* — the parts of a star rating — work.
+
+Most of star rating is derived from *strain* skills, whose values are estimated by splitting a map into multiple sections. Strain for the very first section is calculated only by how the objects in it are positioned within time and space, but every section afterwards combines its own strain with a portion of the strain from the last section.
+
+This is called "residual strain", which gradually decays over time so that objects hit a minute ago do not keep increasing the strain in the present moment.
+
+This process creates a list of "peak strains" for each portion of the map, which are then combined into a final skill value by calculating a weighted sum of all peaks. For instance, the highest peak has a weight of 1, the second-highest — a weight of 0.9, the third highest — a weight of 0.9<sup>2</sup> = 0.81, and so on.
+
+As it turns out, this weighting process means that the highest few peaks constitute most of the final difficulty value for a given skill, which favoured maps with short strain spikes.
+
+That's where [**emu1337**](https://osu.ppy.sh/users/2185987) stepped in, and proposed [a change](https://github.com/ppy/osu/pull/13483) that applies a damping factor to the 10 most difficult sections for the aim skill and the 5 most difficult for the speed skill.
+
+This means that the final star rating values will decrease across the board for those skills, but maps with high but short strain spikes will be impacted more than maps with consistent strain values throughout.
+
+You can take a peek at the following graphs to check out how this looks in practice:
+
+![](/wiki/shared/news/2021-07-27-performance-points-star-rating-updates/weighting_comparison_1.png)
+
+![](/wiki/shared/news/2021-07-27-performance-points-star-rating-updates/weighting_comparison_2.png)
+
+A couple of examples of difficulties that will be impacted by this change more than others are [Brian the Sun - Lonely Go! (TV Size) \[Fiery's Extreme\]](https://osu.ppy.sh/beatmapsets/863227#osu/1860433) and [MOMOIRO CLOVER Z - SANTA SAN \[1-2-SANTA\]](https://osu.ppy.sh/beatmapsets/1045600#osu/2185824).
+
+### All rulesets: Fixed strain skills incorrectly reading out time values when rate was modified by mods
```

Comment:
@smoogipoo does this mean we need to run recalculations for taiko/catch as well..?

Reply:
All rulesets have changed slightly, so ideally we'd run for all rulesets. I'll try to find out exactly how much taiko/catch have changed.

Reply:
List of taiko/catch changes >0.1*: https://docs.google.com/spreadsheets/d/17O7Eq-DBH_Aw2cGX3k4JWEIMRsStm9Qb5zNxL5c9Vis/edit?usp=sharing

I'd do a reprocess of both anyway.